css/responsive-laptop.css

```diff
@@ -0,0 +1,186 @@
+/* ----------- Non-Retina Screens ----------- */
+
+@media screen and (min-device-width: 1200px) {
+    .container {
+        width: 960px;
+        margin: auto;
+    }
+
+    .hamburger,
+    .cross {
+        display: none;
+    }
+
+    section {
+        width: 70%;
+        float: left;
+    }
+
+    .column-1 {
+        width: 50%;
+        float: right;
+    }
+
+    .column-2 {
+        width: 50%;
+        float: left;
+    }
+
+    .column {
+        margin-top: 80px;
+    }
+
+    #main-header {
+        text-align: center;
+        height: 100px;
+        position: static;
+        background: none;
+    }
+
+    aside {
+        overflow: hidden;
```

Comment:
There could be more then 1 aside. Don't use overflow, it's better to add a clearfix.